### PR TITLE
Adding support for set with nx, ex and time

### PR DESCRIPTION
**Pull request by ipapapa**
Rob requested to support `String set(String key, String value, String nxxx, String expx, long time)`

### CODE REVIEW

**Comment by robzienert** [comment]
Thanks! Nice to get a view into what's necessary to add client support for maybe next time. :)

File: dyno-jedis/src/main/java/com/netflix/dyno/jedis/DynoJedisClient.java [hunk 1]
```diff
@@ -1344,6 +1339,31 @@ public String execute(final Jedis client, final ConnectionContext state) throws
...
+                @Override
+                public String execute(Jedis client, ConnectionContext state) throws DynoException {
+                    return client.set(key, value, nxxx, expx, time);
+                }
+            });
+        } else {
+            return connPool.executeWithFailover(new CompressionValueOperation<String>(key, OpName.SET) {
+                @Override
+                public String execute(final Jedis client, final ConnectionContext state) throws DynoException {
+                    return client.set(key, compressValue(value, state), nxxx, expx, time);
```
Comment (shailesh33) [inline dyno-jedis/src/main/java/com/netflix/dyno/jedis/DynoJedisClient.java]: We need to check if dynomite has the appropriate parsing logic in place for this command.

Comment (ipapapa) [inline dyno-jedis/src/main/java/com/netflix/dyno/jedis/DynoJedisClient.java]: It does 

````
redis-cli -p 8102 set mykey "hello" ex 100 NX
OK
````

Comment (shailesh33) [inline dyno-jedis/src/main/java/com/netflix/dyno/jedis/DynoJedisClient.java]: Thanks.

**Pull request merged**